modules/cnv-resuming-node-maintenance-cli.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-maintenance-mode.adoc
+
+[id="cnv-resuming-node-maintenance-cli_{context}"]
+= Resuming a node from maintenance mode in the CLI
+
+Resume a node from maintenance mode and make it schedulable again by deleting 
+the `NodeMaintenance` object for the node. 
+
+.Procedure
+
+. Find the `NodeMaintenance` object:
```

Comment:
How do I know that I have the right one? Showing sample output might help.

Reply:
It should output the node name.

@dshchedr - do you have example output for the '$ oc get nodemaintenance` command?

Reply:
$ oc get nodemaintenance
NAME                   AGE
node-maintenance-job   56s

Reply:
Thanks @dshchedr. In a situation where I have multiple nodes in maintenance, would it be best for me to `$ oc describe nodemaintenance <nodemaintenancejob-name>` to confirm that the `nodeName` in the output matches the node I want to bring out of maintenance?
Or is there a better way?

Reply:
Yeah, you are right, you need to add node maintenance job name to this command. The output will be the same:

$ oc get nodemaintenance <nodemaintenancejob-name>
NAME AGE
<nodemaintenancejob-name>   56s

Reply:
@dshchedr - sorry mate, wasn't 100% sure what you meant. I've added an optional step to use `$ oc describe`, which should output the yaml (included in example). Do you mind having a look and letting me know if this is correct?

Reply:
Sorry, my bad. Here the example of output for "describe" command: 

$ oc describe nodemaintenance nodemaintenance-test
Name:         nodemaintenance-test
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  kubevirt.io/v1alpha1
Kind:         NodeMaintenance
Metadata:
  Creation Timestamp:  2019-06-20T19:51:03Z
  Finalizers:
    foregroundDeleteNodeMaintenance
  Generation:        1
  Resource Version:  1064378
  Self Link:         /apis/kubevirt.io/v1alpha1/nodemaintenances/nodemaintenance-test
  UID:               bc7ec28b-9394-11e9-b691-fa163e2189cb
Spec:
  Node Name:  host-172-16-0-29
  Reason:     Test node maintenance
Events:       <none>

And can you please tell me how can I take a look on updated pages?

Reply:
@dshchedr - Wonderful. Thank you!
I've updated that output a tiny bit, and have posted the most recent build here:
http://file.bne.redhat.com/aburden/CNV-docs-preview/migration/cnv_users_guide/
(There's something screwy with the css when I copy the files to that server, so it looks ugly as sin, but it won't affect the actual build. Also the contents links don't work, but you can navigate the files).
If you can give that a quick look and let me know if it's okay, then this PR will be ready to merge.

Reply:
In general everything looks very good for me. Thank you.
 Just one thing - here the full output of "oc describe nodemaintenance ..." command, it looks a little different from your:

<pre>
$ oc describe nodemaintenance nodemaintenance-test
Name:         nodemaintenance-test
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  kubevirt.io/v1alpha1
Kind:         NodeMaintenance
Metadata:
  Creation Timestamp:  2019-06-20T19:51:03Z
  Finalizers:
    foregroundDeleteNodeMaintenance
  Generation:        1
  Resource Version:  1064378
  Self Link:         /apis/kubevirt.io/v1alpha1/nodemaintenances/nodemaintenance-test
  UID:               bc7ec28b-9394-11e9-b691-fa163e2189cb
Spec:
  Node Name:  host-172-16-0-29
  Reason:     Test node maintenance
Events:       <none>
</pre>

I think it's better to update it..

Reply:
Thanks again @dshchedr - updated the output but snipped out the Metadata to highlight `Node Name`.